source/docs/01-introduction.xml

```diff
@@ -26,11 +26,11 @@
       <fileDesc>
          <titleStmt>
             <title level="s">The Music Encoding Initiative Guidelines</title>
-            <title level="a">Introduction to MEI</title>
+            <title level="a">Introduction to <abbr>MEI</abbr></title>
```

Comment:
Personally I wouldn't add `abbr` tags within the header, but that should be decided by the general editors.

Reply:
@music-encoding/guidelines-editors-in-chief opinions?

Reply:
I wouldn’t either, after all it’s the `fileDesc` we’re talking about.